src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -406,6 +406,12 @@ interface RouteData {
 }
 ```
 
+#### `pages` option
```

Comment:
```suggestion
#### `pages` option

**Type:**
```
If this is supposed to be analogous with `dir` option and `route` option, should it not also have the Type sub-heading?

Reply:
Yep. Added it with the type from line 333